tests/conftest.py

```diff
@@ -994,6 +993,47 @@ def postgis_db():
     yield engine
 
 
+@pytest.fixture()
+def postgres_db():
```

Comment:
The postgres / postGIS fixtures are very similar, as is the little snippet of documentation inside them explaining how to run them. Let's merge them together a bit more - hopefully that will help avoid a situation where a contributor is confused about if they should be running postgres tests or postgis tests.

If preferred, you (or I) could do it in a follow up PR

Here's the idea:
```
@pytest.fixture()
def postgres_db():
    """
    Using docker, you can run all PostgreSQL /  PostGIS tests - such as test_postgis_import - as follows:
        docker run -it --rm -d -p 15432:5432 -e POSTGRES_HOST_AUTH_METHOD=trust postgis/postgis
        KART_POSTGRES_URL='postgresql://postgres:@localhost:15432/postgres' pytest -k "postgres OR postgis" --pdb -vvs
    """
    # postgres_db implementation is the same as you have it now
    
@pytest.fixture()
def postgis_db(postgres_db):
  # See postgres_db for help on running PostGIS tests
  # postgis_db just takes postgres_db fixture and makes sure it has postGIS extension, otherwise it skips

@pytest.fixture()
def no_postgis_db(postgres_db):
  # See postgres_db for help on running no_postgis_db tests.
  # Same implementation as you have currently
```

Reply:
Hmm, I actually made the changes in this PR, would you take a look?